notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>Same here</p><p> </p><p>Also: no sensitivity plots? Or am I missing something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>The sensitivity plots are the first ones shown! Right before the rates.</p>

Reply:
I don't see them in the ReviewNB, can you check?